Makefile

```diff
@@ -146,8 +146,12 @@ test: generate fmt vet envtest ## Run tests.
 	rm cover.out.tmp
 
 .PHONY: e2etest
-e2etest: generate local-deploy chainsaw
-	GIT_REF=$(GIT_REF) $(CHAINSAW) test ./e2e
+e2etest: generate local-release local-deploy chainsaw
+    ifeq ($(strip $(E2E_FLAGS)),) ## If E2E_FLAGS not set, run quick e2e tests
+		GIT_REF=$(GIT_REF) $(CHAINSAW) test ./e2e --selector quick
```

Comment:
given we are always including the selector here, can we maybe have ` --selector ${E2E_SELECTOR} ${E2E_FLAGS} ` and do the switches in GHA?